crates/bevy_pbr/src/render/mesh.rs

```diff
@@ -760,6 +760,10 @@ impl SpecializedMeshPipeline for MeshPipeline {
             depth_write_enabled = true;
         }
 
+        if key.contains(MeshPipelineKey::MOTION_VECTOR_PREPASS) {
```

Comment:
Does having motion vector prepass necessarily imply using TAA? That seems like a fragile assumption?

Reply:
Yeah, it was the best I could come up with. It's not a bad assumption though. 

I'm planning on adding pipeline keys to control the different types of shadow filtering, so it should be moot anyways.